lms/djangoapps/django_comment_client/management/commands/export_discussion_participation.py

```diff
@@ -0,0 +1,157 @@
+import csv
+import dateutil
+from datetime import datetime
+from optparse import make_option
+
+from django.core.management.base import BaseCommand, CommandError
+
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+
+from courseware.courses import get_course
+from student.models import CourseEnrollment
+from lms.lib.comment_client.user import User
+import django_comment_client.utils as utils
+
+
+class _Fields:
```

Comment:
@Kelketek on the other hand, having them in root namespace of the module would pollute the namespace. Plus, this class has semantic naming, so it's clear that these are some fields of something: namely fields of result record. So I believe these advantages outweight "Flat is better than nested" :)